game/tests.py

```
@@ -182,6 +182,26 @@ def test_email_failure_renders_error_and_removes_pending_user(self):
         self.assertNotIn('registration_user_id', self.client.session)
         self.assertNotIn('registration_otp_hash', self.client.session)
 
+    def test_duplicate_email_registration_fails(self):
+        User.objects.create_user(
+            username='existinguser',
+            email='duplicate@example.com',
+            password='StrongPass123!',
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)